pkg/storage/storage.go

```diff
@@ -90,12 +90,14 @@ func getPlatformStorage() (imageregistryv1.ImageRegistryConfigStorage, error) {
 		return cfg, err
 	}
 
-	switch {
-	case installConfig.Platform.Libvirt != nil:
+	switch installConfig.Platform.Name() {
+	case "libvirt":
 		cfg.EmptyDir = &imageregistryv1.ImageRegistryConfigStorageEmptyDir{}
-	case installConfig.Platform.AWS != nil:
+	case "aws":
 		cfg.S3 = &imageregistryv1.ImageRegistryConfigStorageS3{}
-	case installConfig.Platform.OpenStack != nil:
+	case "azure":
+		cfg.PVC = &imageregistryv1.ImageRegistryConfigStoragePVC{}
```

Comment:
i think this needs to be emptydir, unless you are guaranteeing read/write/many PVs will be available on the Azure environment when the registry is created.  As it is you will mostly still hang the install waiting for the registry to be available because the registry won't be able to claim a PV successfully.

Reply:
EmptyDir sounds good to me. Didnt realize that it requires ReadWriteMany PVC, azure default storage class creates a AzureDisk which is just ReadWriteOnce mode. 

```

Volume Plugin | ReadWriteOnce | ReadOnlyMany | ReadWriteMany
-- | -- | -- | --
AWSElasticBlockStore | ✓ | - | -
AzureFile | ✓ | ✓ | ✓
AzureDisk | ✓ | - | -

```

I think blob store matches the requirement more than AzureFile/Disk. Sending an update to use EmptyDir and change later based on requirements (also emptydir is the least risk option right now)